unidist/cli/backends/mpi/runner.py

```diff
@@ -109,7 +111,21 @@ def get_worker_command(num_cpus):
                 ),
             ]
 
-        command += [hosts_str] + command_executor + [":"] + command_monitor
+        if sys.platform == "win32":
+            import mpi4py
+
+            if mpi4py.get_config()["libraries"] == "impi":
+                command += hosts + command_executor + [":"] + command_monitor
+            else:
+                if len(self.hosts) == 1 and self.hosts[0] == get_localhost_ip():
+                    command += command_executor + [":"] + command_monitor
+                else:
+                    raise RuntimeError(
```

Comment:
What if the user passed an IP address of the node instead of `localhost` string? Why should we raise `RuntimeError`? Can we use that address to pass in to mpiexec command?

Reply:
No, we can't. Microsoft MPI mpiexec can't use option `-hosts` without activating separate Windows services. We can't include activating this services in CLI flow. By this reason we require from user to install mpi4py from `intel` conda channel in this case.